target/linux/x86/base-files/lib/preinit/43_grow_rootfspart

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+
+get_squashfs_size() {
+	local low high
+	low=$(hexdump -s 40 -n 4 -e '1/4 "%08x" "\n"' "$1")
+	high=$(hexdump -s 44 -n 4 -e '1/4 "%08x" "\n"' "$1")
+	echo $((0x$high$low))
+}
+
+run_grow_root_part() {
+    [ "$FAILSAFE" != "true" ] || return
```

Comment:
[ "$FAILSAFE" = "true" ] && return
?

Reply:
I'm skipping the growfs if failsafe was trigged. I copied from 40_run_failsafe_hook.
Should I check for /tmp/.failsafe instead?

Reply:
I just wonder why this is "not equal"->OR where "equal"->AND looks simpler to me. Just about the logic inversion ...

Reply:
I'll change it if it looks better.

Reply:
Changed all [!=] || to [=] &&